src/litserve/server.py

```diff
@@ -123,7 +128,10 @@ def run_batched_loop(lit_api, request_queue: Queue, request_buffer, max_batch_si
                 with contextlib.suppress(BrokenPipeError):
                     pipe_s.send((y_enc, LitAPIStatus.OK))
         except Exception as e:
-            logging.exception(e)
+            logger.exception(
+                "Error occurred while processing batched requests.\n"
+                "Hint: Please ensure your LitAPI implementation is correct."
```

Comment:
Can we be more informative for the user here?

Reply:
updated it to ```logger.exception(
                "LitAPI ran into an error while processing the batched request.\n"              
                "Please check the error trace for more details."
            )```

Failure here means user's litapi implementation had issues and error trace would make that clear.